### ISSUE TITLE
tfjs-node: progress bar callback doesn't show enough decimal places for small loss values

### ISSUE THREAD

**Issue by caisq** [tensorflow/tfjs#899]
Currently the `ProgbarCallback`, which is activated by default for tf.Model.fit() call in tfjs-node, shows only two decimal places. This leads to uninformative printouts such as "loss=0.00, val_loss=0.00" when the loss values are small.
